classes/log/store.php

```diff
@@ -136,8 +175,7 @@ public function get_max_batch_size() {
      */
     private function get_persistent_eventids(array $events) {
         foreach ($events as $event) {
-            $eventid = $this->get_event_id($event);
-            $event->logstorestandardlogid = $eventid;
```

Comment:
@lzabo This will fail unless if the $event is an array and not an object.

Reply:
I forget to add the convert.